index.bs

```diff
@@ -460,6 +460,16 @@ Expectations and Discipline</h5>
 	“Disclosure”
 	in the W3C Patent Policy [[!PATENT-POLICY]].
 
+	A [=Chair=] <em class="rfc2119">may</em> protect their group
+	by temporarily suspending any individual
+	from any meeting or discussion under their jurisdiction
```

Comment:
I suggest using another word than "jurisdiction", to avoid introducing legal terminology.

Reply:
supervision?

Reply:
"Chairing" is good, thank you Nigel!